### PR TITLE
update_rhv_glossary preliminary updates to RHV glossary

### PR DESCRIPTION
**Pull request by emarcusRH**
Updates to RHV glossary  
- removing terms that don't exist anymore in the UI or the documentation set
- updating links to documents
- removing examples with links to illustrations that are no longer in the doc det

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
@emarcusRH Thanks for opening this! Just a few questions/comments

**Comment by emarcusRH** [comment]
@bergerhoffer  ready for final review and merge if all looks ok

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc]: ack on singular notification. Thanks!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:81]: This bullet isn't rendering properly since there isn't a blank line above it

**Review by bergerhoffer — COMMENTED** [review]
Thanks for the updates @emarcusRH! Just one small formatting issue, then this will LGTM!

But we require 3 approvals from our project members before merging. @redhat-documentation/ccs-style-council Please review!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (joaedwar) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:58]: I think using the word "shows" might not be recommended. I think 'displayed' might be a good alternative (for translation reasons). WDYT?

```*Description*: The details view displays detailed information about a selected item.
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (joaedwar) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:161]: The first sentence is a bit ambiguous. How about _Always spell out the full product name of the host and do not capitalize the term "host"_?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (joaedwar) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:187]: Maybe something like do not use "the Host" or capitalize the term "host" when it is not used with the full product name.. or something like that. WDYT?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (joaedwar) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:346]: Instead of capital P, maybe something like |"Always use "VM Portal" and capitalize the product name." WDYT?

**Review by joaedwar — APPROVED** [review]
Hi @emarcusRH ! Thanks for this PR. I added a few minor recommendations/ style edits. Thanks a million

**Comment by joaedwar** [comment]
Hi @[ccssytle concil](https://github.com/orgs/redhat-documentation/teams/ccs-style-council). If anyone is available to review and approve this PR, it would be greatly appreciated. Thanks a million

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:47]: We could avoid this 'etc.' (IBM style no-no) by explaining the change of use of the term. Could do something like:

`Use "Data Warehouse", not "the Data Warehouse", unless "Data Warehouse" functions as an adjective, for example, "the Data Warehouse package" or "the Data Warehouse service".`

 The 'for example' would remove the need for an etc.' or an 'and so on' too.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:99]: If `Red Hat Virtualization Host` is singular here, does it need an indefinite article 'a' before it?

Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:99]: Or maybe it should be 'Hosts' plural? Or maybe it's fine :) just getting tripped up on 'two types' then the singular-plural disagreement reads weirdly.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:159]: This is a bit nitpicky but you could maybe make this sentence active and more user oriented with something like:

`You can use Red Hat Enterprise Linux servers that are subscribed to the appropriate entitlements as hosts in Red Hat Virtualization.`

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:187]: On this line it'd be easy to make this passive sentence active:

`You can use "RHVH" in subsequent instances.`

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:213]: Maybe this one could be prescriptive instead of the 'you can', which seems a bit optional:

`Use the name of the tab when you refer to it, for example, "the *Storage* tab".`

something like this would remove the 'just' too, which we should avoid unless it's necessary.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc:296]: Suggestion to make this first sentence active:

*Description*: Use "Standalone Manager" specifically, and only, in the context of differentiating between a "regular" Red Hat Virtualization environment and a self-hosted engine environment.

**Comment by ndeevy** [comment]
Review done @emarcusRH 
A few minor wording type comments but nothing wrong really. Good joerb :D

**Review by joaedwar — APPROVED** [review]
lgtm

**Pull request merged**
